lib/active_admin_import/import_result.rb

```diff
@@ -28,5 +28,11 @@ def empty?
       total == 0
     end
 
+    def failed_message
+      failed.map{|record|
+        errors = record.errors
+        (errors.full_messages.zip errors.keys.map{|k| record.send k}).map{|ms| ms.join(' - ')}.join(', ')
```

Comment:
Line is too long. [105/80]<br>Space missing to the left of {.<br>Space between { and | missing.<br>Space missing inside }.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.